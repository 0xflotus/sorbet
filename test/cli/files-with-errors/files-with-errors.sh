#!/bin/bash
main/sorbet -p error-files -q test/cli/files-with-errors/with-errors.rb test/cli/files-with-errors/without-errors.rb
