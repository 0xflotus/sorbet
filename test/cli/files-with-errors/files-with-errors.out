test/cli/files-with-errors/with-errors.rb
